x-pack/plugins/ml/server/models/data_recognizer/modules/sample_data_ecommerce/ml/high_sum_total_sales.json

```diff
@@ -0,0 +1,38 @@
+{
+  "groups": ["kibana_sample_data"],
+  "description": "Kibana sample eCommerce data: detect anomalies in total sales",
+  "analysis_config": {
+    "bucket_span": "1h",
+    "detectors": [
+      {
+        "detector_description": "Mean total sales",
```

Comment:
Should this just be `Total sales` or `High sum total sales`, rather than `Mean`?

Reply:
Yep, good catch! Fixed to `High total sales` to highlighted the fact that we're trying to detect above avg values.